scripts/release/debian/deb_install.sh

```diff
@@ -42,12 +42,15 @@ setup() {
     set -v
     export DEBIAN_FRONTEND=noninteractive
     apt-get update
-    apt-get install -y apt-transport-https lsb-release gnupg curl
+    apt-get install --assume-yes --no-install-recommends apt-transport-https ca-certificates curl gnupg lsb-release
```

Comment:
I agree to add `--no-install-recommends` to avoid add unnecessary packages, which has been applied to some dockerfiles.

To confirm which packages are affected, I created this script.
The final image has the same installed packages in Ubuntu 22.04.

``` bash
echo "set -ex
apt update &&  apt install -y curl ca-certificates --no-install-recommends
curl -sL https://aka.ms/InstallAzureCLIDeb | bash
apt list --installed | wc
apt list --installed | wc" > old.sh

docker run --rm  -v ./old.sh:/old.sh ubuntu:22.04 bash -c "bash old.sh"
145     603   10186

----------

echo "set -ex
apt update &&  apt install -y curl ca-certificates --no-install-recommends
curl  https://raw.githubusercontent.com/Azure/azure-cli/417bfff384c78fbc4b08b41f5c688c45d19368fa/scripts/release/debian/deb_install.sh | bash
apt list --installed | wc
apt list --installed | wc" > new.sh

docker run --rm  -v ./new.sh:/new.sh ubuntu:22.04 bash -c "bash new.sh"
    145     603   10186
```